zellij-server/src/screen.rs

```diff
@@ -1187,7 +1187,7 @@ impl Screen {
                 tab.visible(false).with_context(err_context)?;
             }
         }
-        if self.active_tab_indices.contains_key(&client_id) {
+        if self.active_tab_indices.len() > 1 && self.active_tab_indices.contains_key(&client_id) {
```

Comment:
Small nitpick here: there is no point to check if the key is in the map before removing it. You can just remove it right away.

Reply:
Remove here & in a couple of other places.